fe/src/main/java/org/apache/doris/analysis/ShowDataStmt.java

```diff
@@ -52,14 +52,16 @@
     private static final ShowResultSetMetaData SHOW_TABLE_DATA_META_DATA =
             ShowResultSetMetaData.builder()
                     .addColumn(new Column("TableName", ScalarType.createVarchar(20)))
-                    .addColumn(new Column("Size", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("DataSize", ScalarType.createVarchar(30)))
```

Comment:
Plz do not modify the origin column name "Size", it may cause some compatibility problem.